src/newHome/mutations.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from "@apollo/client";
+
+export const updateUserMetadata = gql`
+  mutation UpdateUserMetadata($id: ID!, $input: [MetadataInput!]!) {
```

Comment:
this is used only for onboarding so you can call it eg "SaveOnBoardingState", update metadata sounds like generic solution that can be used anywhere